app/presenters/organisations/show_presenter.rb

```diff
@@ -0,0 +1,59 @@
+module Organisations
+  class ShowPresenter
+    attr_reader :org
+
+    def initialize(organisation)
+      @org = organisation
+    end
+
+    def prefixed_title
+      prefix = needs_definite_article?(org.title) ? "the " : ""
+      (prefix + org.title)
+    end
+
+    def latest
+      stories = []
```

Comment:
Not sure this is a good variable name when the section displays the content it's meant to (i.e: not just news stories). Maybe `documents`. Same for here: https://github.com/alphagov/collections/pull/690/files#diff-ce57b95cdb3de98ec1f5711ba7dc9f24R17

On a related note, might be worth updating the Trello and PR to reflect that this section is not related to supergroups any more.